src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenResendInvite.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.divorce.citizen.event;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.divorce.citizen.notification.ApplicationSentForReviewNotification;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingApplicant2Response;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CREATOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.validation.ValidationUtil.validateCitizenResendInvite;
+
+@Slf4j
+@Component
+public class CitizenResendInvite implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String CITIZEN_RESEND_INVITE = "citizen-resend-invite";
+
+    @Value("${applicant_2.to_link_to_case_offset_days}")
+    private int toLinkToCaseOffsetDays;
+
+    @Autowired
+    private ApplicationSentForReviewNotification applicationSentForReviewNotification;
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+
+        configBuilder
+            .event(CITIZEN_RESEND_INVITE)
+            .forStates(AwaitingApplicant2Response)
+            .name("Update applicant 2 email")
+            .description("Citizen event for applicant 1 to update applicant 2 email pre-submission")
+            .aboutToSubmitCallback(this::aboutToSubmit)
+            .grant(CREATE_READ_UPDATE, CREATOR);
```

Comment:
I only want this event for citizen applicant 1 - is this the correct grant permission? @nitinprabhu

Reply:
yes that's correct @danielbe11